packages/unix_home.rb

```diff
@@ -0,0 +1,50 @@
+require 'package'
+
+class Unix_home < Package
+  description 'Regular Unix home directory for Chrome OS'
+  version '1.0'
+  compatibility 'all'
+  license 'GPL-3+'
+  source_url 'file:///dev/null'
+  source_sha256 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'
+  
+  def self.install
+    @HOME = "#{CREW_DEST_PREFIX}/home/#{USER}/"
+    FileUtils.mkdir_p [@HOME, "#{CREW_DEST_PREFIX}/etc/env.d/"]
+    
+    FileUtils.ln_s Dir["#{HOME}/.bash*"], @HOME
+    FileUtils.ln_s "#{HOME}/Downloads", @HOME
+    FileUtils.ln_s "#{HOME}/MyFiles", @HOME + 'Documents'
+    FileUtils.ln_s HOME, @HOME + 'chromeos'
+    
+    @env = <<~EOF
+      set -a
+      HOME=#{@HOME}
+      ORIG_HOME=#{HOME}
+      set +a
+    EOF
+    File.write("#{CREW_DEST_PREFIX}/etc/env.d/unix_home", @env)
+    # add original home directory to const.rb
+    system "echo 'ORIG_HOME=#{HOME}' >> #{CREW_LIB_PATH}/lib/const.rb"
```

Comment:
I don't think appending is a good idea.  Let's just set `ORIG_HOME` if the directory `#{CREW_DEST_PREFIX}/home/#{USER}/` exists and commit `const.rb`.